data_steward/validation/participants/participant_validation_queries.py

```diff
@@ -1,6 +1,28 @@
 # Project imports
 from common import JINJA_ENV
 
+CREATE_NAME_COMPARISON_FUNCTION = JINJA_ENV.from_string("""
+    CREATE FUNCTION IF NOT EXISTS `{{project_id}}.{{drc_dataset_id}}.CompareName`(rdr_name string, ehr_name string)
+    RETURNS string
+    AS ((
+        WITH normalized_rdr_name AS (
+            SELECT REGEXP_REPLACE(LOWER(TRIM(rdr_name)), '[^A-Za-z]', '') AS rdr_name
```

Comment:
@jp3477 this comparison meets the criteria within the scope of the ticket, but double-check with the name_matching in the existing implementation so that we are not missing any edge cases for eg. normalizing the names, etc.

Reply:
I confirmed that this function has the same functionality as the existing [python function](https://github.com/all-of-us/curation/blob/develop/data_steward/validation/participants/normalizers.py#L226).